libs/wire-api/src/Wire/API/Routes/Internal/Galley.hs

```diff
@@ -94,6 +94,8 @@ type IFeatureAPI =
     :<|> IFeatureStatusLockStatusPut AppsConfig
     :<|> IFeatureStatusLockStatusPut SimplifiedUserConnectionRequestQRCodeConfig
     :<|> IFeatureStatusLockStatusPut StealthUsersConfig
+    :<|> IFeatureStatusLockStatusPut MeetingConfig
```

Comment:
if meetings is unlocked by default maybe we should consider to not enable the modification of the lock status at all because there is no requirement from the business perspective. (why would you ever want to lock it)